daemon/graphdriver/windows/windows.go

```diff
@@ -412,7 +412,24 @@ func (d *Driver) Put(id string) error {
 }
 
 // Cleanup ensures the information the driver stores is properly removed.
+// We use this opportunity to cleanup any -removing folders which may be
+// still left if the daemon was killed while it was removing a layer.
 func (d *Driver) Cleanup() error {
+	// Note we don't return an error on cleanup failure, but do log a warning.
+	items, err := ioutil.ReadDir(d.info.HomeDir)
+	if err != nil {
+		return nil
+	}
+	for _, item := range items {
+		if item.IsDir() && strings.HasSuffix(item.Name(), "-removing") {
+			if err := hcsshim.DestroyLayer(d.info, item.Name()); err != nil {
+				logrus.Warnf("Failed to cleanup %s: %s", item.Name(), err)
+			} else {
+				logrus.Infof("Cleaned up %s", item.Name())
```

Comment:
Wondering if this is "info" or "debug" information. Looking for other cleanup actions, those seem to use `Debug`;
https://github.com/docker/docker/blob/7c62f58b27f98140ff3d39de166bf2e107e84a65/plugin/blobstore.go#L69
https://github.com/docker/docker/blob/f67d4b897adc9048269d1b91435b6dcbf69e36d1/libcontainerd/container_windows.go#L61

Open to input what would be best 👍